packages/gatsby-remark-autolink-headers/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.1.20](2019-12-02)
+
+Allow `after` option where the link will appear inline after the header text
+
```

Comment:
Changelog is autogenerated from commit messages, so please don't pre-emptively add to it (it can be edited afterwards if needed)